docker-compose.yml

```
@@ -152,6 +153,30 @@ services:
       args:
         BASE_IMAGE: 'sentry-onpremise-local'
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
+  reverse_proxy:
+    ports:
+      - '9000:80/tcp'
+    image: "nginx:1.16.1"
+    volumes:
+      - type: bind
```

```suggestion
      - type: bind
        read_only: true
```

Let's make the config folder read-only.

---

ok